assets/app/features/upgrade/templates/landing-page.jade

```diff
@@ -1,39 +1,69 @@
-                    active: upgradeLandingVm.prechecks.running\
-                }",
+                        'spinner-visible': upgradeLandingVm.prechecks.spinnerVisible,\
+                        active: upgradeLandingVm.prechecks.running\
+                    }",
                 ng-click="upgradeLandingVm.prechecks.runPrechecks()"
-            )
-                suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prechecks.running",
+                )
+                    suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prechecks.running",
                     visible="upgradeLandingVm.prechecks.spinnerVisible")
-                span(translate='') upgrade.steps.landing.prechecks.form.check-again
-
-    .row
-        .col-sm-10.col-sm-offset-1.note-message.text-center
-            strong(translate='') upgrade.steps.landing.note
+                    span(translate='') upgrade.steps.landing.prechecks.form.check-again
 
-    button.btn.btn-success.center-block(
-        ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running",
-        ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
-        ng-click="upgradeLandingVm.beginUpgrade()"
-    )
-        suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
-            visible="upgradeLandingVm.prepare.spinnerVisible")
-        span(translate='') upgrade.steps.landing.form.button
+            .upgrade.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid, 'inactive': !upgradeLandingVm.mode.valid}")
+                crowbar-circle(ng-class="{'active': upgradeLandingVm.mode.valid}") 3
+                .h4.text-center(translate='') upgrade.steps.landing.upgrade.title
+                p(translate='') upgrade.steps.landing.upgrade.description
+                button.btn.btn-primary(
+                    ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running || !upgradeLandingVm.mode.valid",
+                    ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
+                    ng-click="upgradeLandingVm.beginUpgrade()"
+                )
+                    suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
+                    visible="upgradeLandingVm.prepare.spinnerVisible")
+                    span(translate='') upgrade.steps.landing.form.button
+        .col-md-6
+            .mode.pull-left(ng-class="{'active': upgradeLandingVm.mode.active, 'inactive': !upgradeLandingVm.mode.valid}")
+                crowbar-circle(ng-class="{'active': upgradeLandingVm.mode.active}") 2
+                .h4.text-center(translate='') upgrade.steps.landing.mode.title
+                .initial
+                    p(
```

Comment:
umm, good point on the ng-if, I did not know there was such a huge difference between both methods.